Mekasuit render fix backport.

KostromDan opened the pull request:
Fixes https://github.com/Asek3/Oculus/issues/170

KostromDan committed:
Mekasuit render fix backport. fixes https://github.com/Asek3/Oculus/issues/170

KostromDan committed:
Fixed tabulations and spaces.

KostromDan commented:
![image](https://github.com/user-attachments/assets/f017013f-9bcc-4412-bc31-2fbd2f6f99a1)

dima-dencep commented on src/main/java/net/coderbot/iris/pipeline/newshader/ShaderKey.java:
```diff
@@ -58,22 +58,25 @@ public enum ShaderKey {
+	SHADOW_TEXT           (ProgramId.Shadow,       		AlphaTests.NON_ZERO_ALPHA,  IrisVertexFormats.ENTITY                       ,FogMode.OFF,          LightingModel.LIGHTMAP  ),
+	SHADOW_TEXT_INTENSITY (ProgramId.Shadow,       		AlphaTests.NON_ZERO_ALPHA,  IrisVertexFormats.ENTITY                       ,FogMode.OFF,          LightingModel.LIGHTMAP  ),
+	MEKANISM_FLAME_SHADOW (ProgramId.ShadowEntities,	AlphaTests.ONE_TENTH_ALPHA,	DefaultVertexFormat.POSITION_TEX_COLOR, 		FogMode.OFF, 		  LightingModel.LIGHTMAP  );
+
```
Revert

dima-dencep replied:
Oops, I accidentally highlighted an extra line (the last one with `MEKANISM_FLAME_SHADOW`), it doesn't need to be rolled back

KostromDan committed:
Fixed inconsistent tabulation length between GitHub and IDEs. Replaced tabs with spaces to ensure consistent formatting across environments.

KostromDan commented:
Noticed differences in tab length between GitHub and IDEs. If it looked correct in the IDE, it looked wrong on GitHub. When I fixed it in the GitHub editor and copied it back to the IDE, it started looking incorrect there. So, I replaced tabs with spaces, and now it looks correct in both places.

dima-dencep reviewed (changes requested):
I hate to break it to you, but you made even more unnecessary changes

dima-dencep commented:
> Noticed differences in tab length between GitHub and IDEs

skill issue

KostromDan commented:
> > Noticed differences in tab length between GitHub and IDEs
> 
> skill issue

https://github.com/Asek3/Oculus/blob/1.19.2/src/main/java/net/coderbot/iris/pipeline/newshader/ShaderKey.java#L63
Wdym, is it correct? In IntelliJ IDEA or VS Code it looks correct, but not on GitHub.
How it looks on GitHub:
![image](https://github.com/user-attachments/assets/a77bc847-67cc-4ecd-86c1-aafd8f90de85)
How it looks in IntelliJ IDEA(tabs are highlited):
![image](https://github.com/user-attachments/assets/9676b3f1-cf59-4df4-9ded-d75618445de4)

KostromDan commented:
If you want I'll revert my fixes and apply only requested, but I think it should look correct on both IDEs and GitHub.

dima-dencep commented:
I don't care how it looks to you, you can see how it looks to everyone in your own pr and I don't understand why you don't see these changes in diff

KostromDan committed:
Revert "Fixed inconsistent tabulation length between GitHub and IDEs. Replaced tabs with spaces to ensure consistent formatting across environments."

This reverts commit 3a0a3005943672e6e48567dbebeff43334eaa7cb.

KostromDan committed:
Revert "Fixed tabulations and spaces."

This reverts commit ded724e59f2ca179ff4ed4f18eaddc1d649c1b3f.

KostromDan committed:
Revert formatting for ShaderKey.
